test/model_test.dart

```diff
@@ -2757,10 +2760,9 @@ String topLevelFunction(int param1, bool param2, Cool coolBeans,
 
     test('has extended documentation', () {
       expect(lengthX.hasExtendedDocumentation, isTrue);
-      expect(
-          lengthX.oneLineDoc,
-          equals(
-              'Returns a length. <a href="fake/WithGetterAndSetter/lengthX.html">[...]</a>'));
+      expect(lengthX.oneLineDoc, equals('Returns a length.'));
+      expect(lengthX.extendedDocLink,
+          equals(ModelElementRendererHtml().renderExtendedDocLink(lengthX)));
```

Comment:
I think we should have at least one literal matching string here for the Html renderer test, especially since it is very targeted -- otherwise we're simply validating that extendedDocLink returns whatever the renderer does.

Reply:
Done